### ISSUE TITLE
Make a determination on Snap Packaging

### ISSUE THREAD

**Issue by tonybalandiuk** [o3de/sig-release#178]
Raised by @amzn-changml  in 4/20 release prep meeting. 
We need to determine if Snap Packaging will make it or not.
@chanmosq  , @tonybalandiuk , Nicole H need to be informed so that we can account for it in release notes, announcement, etc.

**Comment by amzn-changml** [comment]
To list out the current issues:

- The snap package in its current state as of  [1259429](https://github.com/o3de/o3de/commit/125942965d9ff86e925048caf7b7a80aa6aeecb8) will not install a working executable. Starting `o3de` will result in a error loading `libQt5Concurrent.so.5`. This is filed as a bug here: https://github.com/o3de/o3de/issues/15581 and to be fixed with this PR: https://github.com/o3de/o3de/pull/15735

- There is a secondary issue with the project manager. There is an error detecting and pulling Python into the O3DE snap folder inside /snap/ (which probably shouldn't be writable). A bug issue is being made and I'll edit this post with the link here

- Finally, the snap upload and publish process is being blocked by a Canonical owned Automated Review (AR). When we push the package, it fails the AR due to the build node running out of space. We can't control the infrastructure as it is owned by Canonical, so they will have to manually review the package if they can't fix this.

**Comment by vincent6767** [comment]
Do you know what's the next action item for this? @Naomi-Wash @tonybalandiuk

**Comment by tonybalandiuk** [comment]
Overall, a point release is something that SIG-Release would make the call on and coordinate, but we would need input from the rest of the project on:
A) whether we should have a point release
B) do we have volunteers create the point release

We don't have a documented process for a point release - we haven't had to do one yet. They have to happen at some point, though, and the process would basically be a subset of the major release steps.

**Comment by Ulrick28** [comment]
Snap packaging should not need a point release, unless we have to make code changes to O3DE itself, which we shouldn't for packaging.

However, there is talk of a point release for other reasons, but that requires a more in depth conversation and data gathering on specific issues to be addressed by a point release.

Joe B.

**Comment by vincent6767** [comment]
Hi @Ulrick28 Are there any updates on this issue? Just wondering whether we have completed this or not.

**Comment by vincent6767** [comment]
Resolved as it's not part of the blockers for 23.05.
